Marlin/Configuration.h

```diff
@@ -118,7 +118,7 @@
 // The following define selects which electronics board you have.
 // Please choose the name from boards.h that matches your setup
 #ifndef MOTHERBOARD
-  #define MOTHERBOARD BOARD_RAMPS_14_EFB
+  #define MOTHERBOARD BOARD_RAMPS4DUE_EFB
```

Comment:
Agreed. The main branch should have a folder `example_configurations/32bit` in the final release. We can have a separate 32-bit branch for now in which this is the main config, and then we might set up Travis CI to only test 32-bit builds in that branch.